pyproject.toml

```
@@ -27,7 +27,7 @@ readme = "README.md"
 requires-python = ">=3.10" # match classifiers above and ruff version below
 
 dependencies = [
-  "numpy>=1.26",  # Required because of our "NPY_TARGET_VERSION"
+  "numpy>=1.26",
```

Do we even need to require a minimum NumPy version anymore?

---

Nope, but I don't know what our actual version lower bound is, it's probably `1.21.2` which was the first version to add 3.10 support. We can certainly remove this.

```suggestion
  "numpy",
```

---

We'll see how this goes, worst case we tell users to upgrade, then add a constraint back